t/configuration-loading-boot-with-config.t

```diff
@@ -1,10 +1,8 @@
 use lib 't';
 use Test::APIcast::Blackbox 'no_plan';
 
-$ENV{APICAST_CONFIGURATION_LOADER} = 'boot';
-
-env_to_nginx(
-    'APICAST_CONFIGURATION_LOADER',
+env_to_apicast(
+    'APICAST_CONFIGURATION_LOADER' => 'boot'
```

Comment:
Was this allowed before or it's supported since Test::APIcast 0.08 ?

Reply:
This wasn't really working before Test::APIcast 0.08. `env_to_nginx` was not working in Test::APIcast::Blackbox at all.